src/DotVVM.Framework.Hosting.AspNetCore/ServiceCollectionExtensions.cs

```diff
@@ -13,6 +13,7 @@ public static class ServiceCollectionExtensions
         /// Adds DotVVM services with authorization and data protection to the specified <see cref="IServiceCollection" />.
         /// </summary>
         /// <param name="services">The <see cref="IServiceCollection" /> to add services to.</param>
+        // ReSharper disable once InconsistentNaming
         public static IServiceCollection AddDotVVM<TServiceConfigurator>(this IServiceCollection services) where TServiceConfigurator : IDotvvmServiceConfigurator, new()
         {
             AddDotVVMServices(services);
```

Comment:
Resharper is right in this case, we usualy use names like `AddDotvvmServices`

Reply:
The method name is same as long as we added support for dotnet core.  I just added overload. I don't think it is good idea to change the name. What about `UseDotVVM`? That's the same case.

Reply:
no we can't touch the public api and it's not against the naming convention if `DotVVM` is not followed by any other text. I was looking at `AddDotVVMServices`, just misplaced the comment, sorry.